src/components/diff-view.jsx

```diff
@@ -27,13 +27,13 @@ import ChangesOnlyDiff from './changes-only-diff';
 export default class DiffView extends React.Component {
   constructor (props) {
     super(props);
-    this.state = {diff: null};
+    this.state = {diffData: null};
```

Comment:
Is this to try and disambiguate the envelope with the diff + its metadata from the diff it contains? I wonder if maybe we should rename the `diff` property to `changes` or something (which would happen on the `-processing` side). Any thoughts on that, @danielballan?

Not really a big deal, though.

Reply:
I'm fine renaming something on teh -processing side if it makes this easier. We should probably avoid `changes` since one might expect that to hold a list of `Change` records or their IDs.

Reply:
> Is this to try and disambiguate the envelope with the diff + its metadata from the diff it contains?

Yea, for me I see `diff` as the html returned from the differs but this property contains the diff + metadata, so I wanted to make that clearer.

Reply:
Probably best if we keep this as-is then and make no changes on the processing side.